src/pretalx/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -4554,10 +4575,6 @@ msgstr "Composer des courriels"
 msgid "Sent E-mails"
 msgstr "Courriels envoyés"
 
-#: pretalx/orga/templates/orga/base.html:404
-msgid "Organisers"
-msgstr "Organisateurs"
-
```

Comment:
Why is this being removed? Isn't this still relevant for this: https://github.com/fossasia/eventyay-talk/blob/297c53ef94c98c28ce195fe16a9a23f5e974b2b7/src/pretalx/orga/templates/orga/base.html#L404?

Reply:
Hi! It is not removed but just moved up in the file.
That is automated. Probably translations changed somewhere but the file was not updated then.
Same applies for other messages as well.

![image](https://github.com/fossasia/eventyay-talk/assets/34372791/118194b3-61a4-4f75-a43c-7989392de9c4)